avionics/statemachine.cpp

```diff
@@ -198,6 +199,7 @@ void stateMachine(float *altitude, float *delta_altitude, float *prev_delta_alti
             }
             if((camera_toggle_count >= TOGGLE_CAMERA_INTERVAL) && (camera_toggle)){
                 stop_record();
+                delay(15);
```

Comment:
Isn't delay in milliseconds??? This should be at least 3 seconds, maybe make it 5 seconds to be safe, which would mean passing 5000 not 15.

Reply:
This is a stop-> start transition, which Elizabeth said she just needed 10 ms for.  A start -> stop needs 3 seconds